docs/pages/enroll-resources/agents/join-services-to-your-cluster/oracle.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Join Services with Oracle Cloud
+description: Use the Oracle join method to add services to your Teleport cluster.
+---
+
+This guide will explain how to use the **Oracle join method** to configure
+Teleport processes to join your Teleport cluster without sharing any secrets
+when they are running in an OCI Compute instance.
+
+The Oracle join method is available to any Teleport process running on an
+OCI Compute instance.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- An OCI Compute instance to host a Teleport service.
+- (!docs/pages/includes/tctl.mdx!)
```

Comment:
Should we also add "A Teleport Cluster running v17.3.0 or later"

Reply:
The edition-prereqs-tabs snippet already has it covered.

Reply:
i don't think it would mention 17.3 specifically though, but see my suggestion i think you can set that variable on the include